pyClient/zeth/joinsplit.py

```diff
@@ -45,6 +52,32 @@
 ComputeHSigCB = Callable[[bytes, bytes, JoinsplitSigVerificationKey], bytes]
 
 
+def blake2s_compress(left: bytes, right: bytes) -> bytes:
+    """
+    Execute blake2s as a compression function, ensuring that the input is of the
+    correct length.  (Note we support len(left) != len(right) as long as the
+    total length is appropriate).
```

Comment:
total length is appropriate -> total input length is appropriate

By appropriate you mean, input length = block size, right?

Reply:
reworded